### PR TITLE
build(Server): Disable Robot instance

### PR DESCRIPTION
**Pull request by GoetzGoerisch**

### CODE REVIEW

**Comment by GoetzGoerisch** [comment]
@ccvca could you please have a look on the remaining linter errors: https://github.com/umati/Sample-Server/pull/257/checks?check_run_id=2575959707#step:4:120

Please review the reformating of https://github.com/umati/Sample-Server/pull/257/commits/b33af12842ad8a527d2a992cb84b91c51331b040.

File: SampleServer.cpp [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```
Comment (ccvca) [inline SampleServer.cpp:1]: Can this be added later in a separate pull request in all files?

File: SampleServer.cpp [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+ *    Copyright 2019-2021 (c) Christian von Arnim, ISW University of Stuttgart  (for VDW and umati)
+ *    Copyright 2020 (c) Tonja Heinemann, ISW University of Stuttgart  (for VDW and umati)
+ *    Copyright 2021 (c) Goetz Goerisch, VDW - Verein Deutscher Werkzeugmaschinenfabriken e.V.
+ */
+
+#include "src_generated/namespace_di_generated.h"
+#include "src_generated/namespace_ia_generated.h"
+#include "src_generated/namespace_machinery_generated.h"
+#include "src_generated/namespace_machinetool_generated.h"
+/*#include "src_generated/namespace_robotics_generated.h"*/
```
Comment (ccvca) [inline SampleServer.cpp:14]: Linter forces a specific ordering of includes, the "internal" ones must be included lastly.

File: SampleServer.cpp (hunk 1 above)
Comment (GoetzGoerisch) [reply SampleServer.cpp]: Yes, dropped.

File: SampleServer.cpp (hunk 2 above)
Comment (GoetzGoerisch) [reply SampleServer.cpp]: fixed

**Review by ccvca — APPROVED** [review]

**Pull request merged**